### DISCUSSION TITLE
How to Install selected package

### DISCUSSION THREAD

**Discussion by rajeevrrs** [modoboa/modoboa#2665] [General]
Hi, 

When i run this code `sudo ./run.py --interactive example.com` it says it will install 11 packages and there is no option to select. It just ask Yes or No. 

I have 1 GB RAM and 1 CPU. when i install whole package. my server getting crashed. How to select? 

I just need

Dovecot 
Modoboa Admin Panel (nginx & mysql)

**Comment by Spitfireap** [comment]
Hi, 
You're looking for ``--stop-after-configfile-check``. Then edit installer.cfg with your need (switch enabled to ``=false`` :)

**Reply by rajeevrrs** [reply]
modoboa 
automx 
amavis 
clamav 
dovecot 
nginx 
razor 
postfix 
postwhite 
spamassassin    
uwsgi 
radicale 
opendkim

This all the packages it install. I need Modoboa, Nginx and Dovecot

**Reply by rajeevrrs** [reply]
If you can says what are mandatory to function Modoboa? 

Like nginx is mandatory. i do not know about rest. 

Postfix is also mandatory for SMTP but i have planned to use external SMTP
